app/views/movies/show.html.erb

```diff
@@ -0,0 +1,22 @@
+<h1> Movie</h1>
+
+<p>Movie Name: <%= @movie.title %></p>
+<p>Year Created: <%= @movie.creation_year %></p>
+<p>Genre: <%= @movie.genre %></p>
+
+<% @movie.actors.each do |actor| %>
+  <p>Actors Name: <%= actor.name %></p>
+  <p>Age: <%= actor.age %></p>
+<% end %>
+
+<p>The average age of each actory is: <%= @movie.average_age %></p>
+
+
+
+<%= form_with(url: "/movies/#{@movie.id}/actors", method: :post) do |form| %>
+  <%= form.label :actor_id%>
+  <%= form.text_field :actor_id%>
+
+
+  <%= form.submit 'Add Actor' %>
+<% end %>
```

Comment:
Great HTML and great implementation all around, I want to just call out that you want to make sure to include `data: { turbo: false }` for your `form_with`.